localPackages/site_developer/Configuration/TCA/Overrides/sys_template.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+(function (string $extensionKey = 'site_developer') {
```

Comment:
Is there some kind of convention how to name the folder with the local packages? (Helmut uses `packages` IIRC.)

Reply:
I'm not aware of any. That's just our convention.